src/core/lightning/index.ts

```diff
@@ -344,9 +345,11 @@ export const LightningMixin = (superclass) =>
           }
 
           // adding contact for the payer
-          if (payeeUser.username) {
-            await addContact({ uid: this.user._id, username: payeeUser.username })
-          }
+          const addContactResult = await addNewWalletContact({
+            userId: this.user.id,
+            contactWalletId: payeeUser.id,
+          })
+          if (addContactResult instanceof Error) throw addContactResult
 
           // adding contact for the payee
           if (this.user.username) {
```

Comment:
If it's ok to delete this no-longer-supported direction here, I can also delete the soon-to-be-redundant `addContact` method